### PR TITLE
feat: DEVOPS-141 service cache function on io-infra migration

### PR DESCRIPTION
**Pull request by diegolagospagopa**
### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [ ] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### How to apply

After PR is approved
1. run deploy pipeline from Azure DevOps [io-infra.deploy](https://dev.azure.com/pagopaspa/io-iac-projects/_build?definitionId=218)
2. select PR branch
3. wait for approval

### CODE REVIEW

**Review by balanza — COMMENTED** [review]
We have never used the naming pattern _XX_name.tf_. I see the value in order to sort files, I just won't introduce such formalism only for a small set of files. Maybe we can propose such refactor in a dedicated PR

File: src/core/env/prod/terraform.tfvars [hunk 1]
```diff
@@ -27,6 +27,7 @@ cidr_subnet_fnpblevtdispatcher = ["10.0.12.0/24"]
 cidr_subnet_appgateway         = ["10.0.13.0/24"]
 cidr_subnet_redis_apim         = ["10.0.14.0/24"]
 cidr_subnet_apim               = ["10.0.101.0/24"]
+cidr_subnet_fn_service_cache   = ["10.0.125.0/24"]
```
Comment (pasqualedevita) [inline src/core/env/prod/terraform.tfvars:30]:
```suggestion
cidr_subnet_fn_service_cache   = ["10.0.129.0/24"]
```

**Review by pasqualedevita — CHANGES_REQUESTED** [review]
LGTM, only rename in services-cache (final s)

File: src/core/env/prod/terraform.tfvars (hunk 1 above)
Comment (pasqualedevita) [reply src/core/env/prod/terraform.tfvars:30]: 10.0.125.0 is the actual subnet and we have a conflict

**Comment by diegolagospagopa** [comment]
> We have never used the naming pattern _XX_name.tf_. I see the value in order to sort files, I just won't introduce such formalism only for a small set of files. Maybe we can propose such refactor in a dedicated PR

Removed files with data, we can think in future to have a xx_name.tf to simplify the use of the project

**Review by pasqualedevita — DISMISSED** [review]

**Pull request merged**